javascript.md

```diff
@@ -63,13 +63,12 @@ JavaScript CodeStyle
     * only once in the current scope;
     * on a new line;
     * as close as possible to the place where it's first used.
-
-  * Each `var` statement should have only one variable declared in it.
+  * Implied global variables should not be used.
 
 **Good:**
 ```javascript
-var keys = ['foo', 'bar'];
-var values = [23, 42];
+var keys = ['foo', 'bar'],
```

Comment:
What's about to provide both declaration options?

**Good**
```javascript
var keys = ['foo', 'bar'];
var values = [23, 42];

var object = {};
while (items.length) {
    var key = keys.pop();
    object[key] = values.pop();
}
```

or 

```javascript
var keys = ['foo', 'bar'],
    values = [23, 42];

var object = {};
while (items.length) {
    var key = keys.pop();
    object[key] = values.pop();
}
```

**Bad**
```javascript
keys = ['foo', 'bar'];
values = [23, 42];

object = {};
while (items.length) {
    key = keys.pop();
    object[key] = values.pop();
}
```